### PR TITLE
Improve verify brooklyn RC script

### PR DESCRIPTION
**Pull request by tbouron**
This does the following things:
- improve the readability of the script
- ignore the NOTICE/LICENSE files check for the DEB package (as there is no reliable way of getting the content of that archive)

### CODE REVIEW

File: release/verify_brooklyn_rc.sh [hunk 1]
```diff
@@ -105,65 +115,74 @@ for ARCHIVE in $(find * -type f ! \( -name '*.asc' -o -name '*.md5' -o -name '*.
       ;;
```
Comment (drigodwin) [inline release/verify_brooklyn_rc.sh]: the line `PREFIX="/opt/brooklyn"` should be `PREFIX="/opt/brooklyn-{RELEASE}"`

Comment (tbouron) [reply release/verify_brooklyn_rc.sh]: `/opt/brooklyn` is a symlink to `/opt/brooklyn-<version>` so it should work all the same. I actually tested it with rc3 and it went fine :)

```
===============================================================================
= Checking LICENSE and NOTICE files in artifacts ...                          =
===============================================================================

/opt/brooklyn-0.12.0/NOTICE
/opt/brooklyn-0.12.0/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-1.noarch.rpm
-rw-r--r--  0 ubuntu ubuntu    169 21 Sep 20:23 apache-brooklyn-0.12.0-bin/NOTICE
-rw-r--r--  0 ubuntu ubuntu 110028 21 Sep 20:23 apache-brooklyn-0.12.0-bin/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-bin.tar.gz
-rw-r--r--  3.0 unx      169 tx      123 defN 17-Sep-21 20:23 apache-brooklyn-0.12.0-bin/NOTICE
-rw-r--r--  3.0 unx   110028 tx    26190 defN 17-Sep-21 20:23 apache-brooklyn-0.12.0-bin/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-bin.zip
-rw-r--r--  0 ubuntu ubuntu    169 20 Sep 16:27 apache-brooklyn-0.12.0-classic/NOTICE
-rw-r--r--  0 ubuntu ubuntu 113732 20 Sep 16:27 apache-brooklyn-0.12.0-classic/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-classic.tar.gz
-rw-r--r--  3.0 unx      169 tx      123 defN 17-Sep-20 16:27 apache-brooklyn-0.12.0-classic/NOTICE
-rw-r--r--  3.0 unx   113732 tx    25814 defN 17-Sep-20 16:27 apache-brooklyn-0.12.0-classic/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-classic.zip
-rw-rw-r--  0 ubuntu ubuntu     169 21 Sep 20:30 apache-brooklyn-0.12.0-client-cli-linux/NOTICE
-rw-rw-r--  0 ubuntu ubuntu  34981 21 Sep 20:30 apache-brooklyn-0.12.0-client-cli-linux/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-client-cli-linux.tar.gz
-rw-rw-r--  3.0 unx      169 tx      123 defN 17-Sep-21 20:30 apache-brooklyn-0.12.0-client-cli-linux/NOTICE
-rw-rw-r--  3.0 unx    34981 tx     7510 defN 17-Sep-21 20:30 apache-brooklyn-0.12.0-client-cli-linux/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-client-cli-linux.zip
-rw-rw-r--  0 ubuntu ubuntu      169 21 Sep 20:30 apache-brooklyn-0.12.0-client-cli-macosx/NOTICE
-rw-rw-r--  0 ubuntu ubuntu  34981 21 Sep 20:30 apache-brooklyn-0.12.0-client-cli-macosx/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-client-cli-macosx.tar.gz
-rw-rw-r--  3.0 unx      169 tx      123 defN 17-Sep-21 20:30 apache-brooklyn-0.12.0-client-cli-macosx/NOTICE
-rw-rw-r--  3.0 unx    34981 tx     7510 defN 17-Sep-21 20:30 apache-brooklyn-0.12.0-client-cli-macosx/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-client-cli-macosx.zip
-rw-rw-r--  0 ubuntu ubuntu    169 21 Sep 20:30 apache-brooklyn-0.12.0-client-cli-windows/NOTICE
-rw-rw-r--  0 ubuntu ubuntu  34981 21 Sep 20:30 apache-brooklyn-0.12.0-client-cli-windows/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-client-cli-windows.tar.gz
-rw-rw-r--  3.0 unx      169 tx      123 defN 17-Sep-21 20:30 apache-brooklyn-0.12.0-client-cli-windows/NOTICE
-rw-rw-r--  3.0 unx    34981 tx     7510 defN 17-Sep-21 20:30 apache-brooklyn-0.12.0-client-cli-windows/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-client-cli-windows.zip
-rw-rw-r--  0 ubuntu ubuntu    169 20 Sep 16:27 apache-brooklyn-0.12.0-src/NOTICE
-rw-rw-r--  0 ubuntu ubuntu  34981 20 Sep 16:27 apache-brooklyn-0.12.0-src/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-src.tar.gz
-rw-rw-r--  3.0 unx      169 tx      123 defN 17-Sep-20 16:27 apache-brooklyn-0.12.0-src/NOTICE
-rw-rw-r--  3.0 unx    34981 tx     7510 defN 17-Sep-20 16:27 apache-brooklyn-0.12.0-src/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-src.zip
-rw-r--r--  0 ubuntu ubuntu    205 20 Sep 16:27 apache-brooklyn-0.12.0-vagrant/NOTICE
-rw-r--r--  0 ubuntu ubuntu  11358 20 Sep 16:27 apache-brooklyn-0.12.0-vagrant/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-vagrant.tar.gz
-rw-r--r--  3.0 unx      205 tx      147 defN 17-Sep-20 16:27 apache-brooklyn-0.12.0-vagrant/NOTICE
-rw-r--r--  3.0 unx    11358 tx     3949 defN 17-Sep-20 16:27 apache-brooklyn-0.12.0-vagrant/LICENSE
[✓] Files LICENSE and NOTICE present in apache-brooklyn-0.12.0-rc3-vagrant.zip
```

Comment (tbouron) [reply release/verify_brooklyn_rc.sh]: Actually scratch that @drigodwin, I updated the script but forgot to push it (which I just did)

**Comment by tbouron** [comment]
retest this please

**Pull request merged**